### DISCUSSION TITLE
Sos

### DISCUSSION THREAD

**Discussion by igor-porche** [Chia-Network/chia-blockchain#14756] [Support]
Help!  sent coins to my wallet and they didn't arrive!  tried several translations and none!  What shall I do?
xch1nrag98f8mfjp8gqjwzgg0njfpm7y3y8mydfqhtyxmkpvknd8vv0s48jkfz

**Comment by shaneo257** [comment]
per https://xchscan.com/address/xch1nrag98f8mfjp8gqjwzgg0njfpm7y3y8mydfqhtyxmkpvknd8vv0s48jkfz it shows in incoming tx from huobi, have you tried to shut down the client, delete the wallet DB, and then restart

**Reply by igor-porche** [reply]
remove wallet ?  and install again?  can you be more detailed?

**Reply by shaneo257** [reply]
after you close the client, delete the files located in \.chia\mainnet\wallet\db, then restart
it'll rebuild your wallet database and you should then see your coins

**Reply by igor-porche** [reply]
deleted the app, now I can't access my wallet!  all words and passwords are entered correctly!
where to find such files on an iPhone?

**Comment by igor-porche** [comment]
deleted the app, now I can't access my wallet!  all words and passwords are
entered correctly!

where to find such files on an iPhone?

чт, 9 бер. 2023 р. о 08:10 shaneo257 ***@***.***> пише:

**Reply by igor-porche** [reply]
? Help

**Comment by igor-porche** [comment]
help me!

чт, 9 бер. 2023 р. о 09:47 Igor Belinskiy ***@***.***> пише:

**Comment by shaneo257** [comment]
hi, sorry, i was on a couple of days off
what app are you using?
if you're using nucle, they are no longer maintaining the node, please use pawket, you can input the same 12 words from nucle to pawket
https://info.pawket.app/

**Comment by igor-porche** [comment]
thank you!  code activation request?  nothing came to the mail 🤷 ♂️ invite
me


нд, 12 бер. 2023 р. о 01:20 shaneo257 ***@***.***> пише:

**Reply by shaneo257** [reply]
if you are now using pawket, it is not maintained by chia, but by a 3rd party, you can reach out to them on discord https://discord.gg/ndQgpHwe4V
